bayeux.go

```diff
@@ -260,19 +254,11 @@ func GetSalesforceCredentials() Credentials {
 	} else if err != nil {
 		logger.Fatal(err)
 	} else if creds.AccessToken == "" {
-		logger.Fatalf("Unable to fetch access token. Check credentials in environmental variables")
+		logger.Fatalf("Unable to fetch access token.")
```

Comment:
Could we please modify this function so that it returns `(*Credentials, error)` and don't log-fatal in it?